genesyscloud/resource_genesyscloud_knowledge_knowledgebase.go

```diff
@@ -169,7 +171,7 @@ func readKnowledgeKnowledgebase(ctx context.Context, d *schema.ResourceData, met
 		cc := consistency_checker.NewConsistencyCheck(ctx, d, meta, ResourceKnowledgeKnowledgebase())
 
 		d.Set("name", *knowledgeBase.Name)
-		d.Set("description", *knowledgeBase.Description)
```

Comment:
sure I can do that @carnellj-genesys , and just to be clear, this will just ensure a NPD doesn't occur, right?. If for instance the name (which is required and expected) is unexpectedly null in the response, what do we expect to happen? Do we care really during this read method?

Reply:
@bbbco @carnellj-genesys After playing around with the API, it looks like name should be required. It's not marked as required in the API explorer, but you get an error back if you omit it.